serverless/cloud_run.tf

```diff
@@ -0,0 +1,48 @@
+resource "google_compute_region_network_endpoint_group" "cloudrun_neg" {
+  name                  = "cloudrun-neg"
+  network_endpoint_type = "SERVERLESS"
+  region                = var.region
+  cloud_run {
+    service = google_cloud_run_service.default.name
+  }
+}
+
+resource "google_cloud_run_service" "default" {
+  name     = "cloudrun"
+  location = var.region
+
+  template {
```

Comment:
có handle được biến môi trường, các cấu hình definition cho container ở đây ko?

Reply:
- Có thể add biến môi trường 
- Cấu hình definition chính là template này ạ, ở gcp gọi là revision 
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_service#revision_name:~:text=The%20template%20block%20supports%3A